ote/src/clj/ote/services/pre_notices/operator.clj

```diff
@@ -16,14 +16,31 @@
 
 (defqueries "ote/services/pre_notices/regions.sql")
 
+(defn- parse-attachments
+  [attachments db user]
+  (let [saved-attachments (mapv
+                            #(first (specql/fetch db ::transit/pre-notice-attachment
+                                           (specql/columns ::transit/pre-notice-attachment)
+                                                  (op/and
+                                                  {::transit/attachment-file-name (::transit/attachment-file-name %)
```

Comment:
Get attachment by `id`, not name... you can reasonably expect there to be files with the same name. Also id, being the primary key, will always be indexed.